client/src/pages/Home.js

```diff
@@ -36,6 +38,16 @@ export function Home() {
 					{message}
 				</h1>
 				<Link to="/about/this/site">About</Link>
+				<Box>
+				<IconButton
+      component="a"
+      href="https://mui.com/material-ui/react-image-list/"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <FacebookIcon style={{ fontSize: 40, color: "black" }} />
```

Comment:
I prefer blue as the facebook colour is blue.

Reply:
we can look at the mockup or check with PO tomorrow. 
well done @karleenmsrichards . first one down... thats half the battle

Reply:
I believe that this is just a test @farzaneh-haghani  hence I paid little mind to the color and design.
Thank you @Tony-devops

Reply:
well done Karleen.Good start.